.devcontainer/devcontainer.json

```diff
@@ -32,11 +32,23 @@
 
 	// Add the locally installed dotnet to the path to ensure that it is activated
 	// This allows developers to just use 'dotnet build' on the command-line, and the local dotnet version will be used.
+	// Add the global tools dir to the PATH so that globally installed tools will work
 	"remoteEnv": {
-		"PATH": "${containerWorkspaceFolder}/.dotnet:${containerEnv:PATH}",
-		"DOTNET_MULTILEVEL_LOOKUP": "0"
+		"PATH": "${containerWorkspaceFolder}/.dotnet:${containerEnv:PATH}:${containerEnv:HOME}/.dotnet/tools",
```

Comment:
looks like `${containerEnv:HOME}` isn't right (it's expanding to `''`).  Maybe `${localEnv:HOME}` is what I want?

but really what I want is to specify a different directory for the global dotnet tools.  Is there some environment variable for that?

Reply:
Decided to install dotnet-serve into `.dotnet-tools-global` in the workspace folder, and add that to the path.